Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py

```diff
@@ -19,7 +19,18 @@
     '2': 'Closed',
     '3': 'Closed'
 }
-INCIDENT_ALIAS = 'alert' if is_xsiam() else 'incident'
+
+
+def get_incident_alias():
+    if is_xsiam():
+        return 'alert'
+    elif is_platform():
+        return 'issue'
+    else:
+        return 'incident'
+
+
+INCIDENT_ALIAS = get_incident_alias()
```

Comment:
Why, in this case, have we decided to fix the lingo and in others not?
@yuvalbenshalom @israelpoli, how does this intersect with your effort?